test/teiserver/battle/balance_lib_test.exs

```diff
@@ -1,6 +1,10 @@
 defmodule Teiserver.Battle.BalanceLibTest do
-  @moduledoc false
+  @moduledoc """
+  Can run tests in this file only by
+  mix test test/teiserver/battle/balance_lib_test.exs
+  """
```

Comment:
I wouldn't put this doc here as it's more something for `mix test` documentation.

Reply:
I did this as a way to have something to copy and paste rather than typing it out by hand each time. Would it be acceptable to put as a `#` comment?